### ISSUE TITLE
Fallback to regular manpage if not found in the tldr cache.

### ISSUE THREAD

**Issue by dflock** [dbrgn/tealdeer#158]
This happens less often than it used to, but it would be nice if tealdeer had an option to fallback to trying the regular `man $1`, if `$1` isn't found in the tldr cache.

```shell
$ tldr --update
Successfully updated cache.

$ tldr look
Page look not found in cache
Try updating with `tldr --update`, or submit a pull request to:
https://github.com/tldr-pages/tldr

$ man look
LOOK(1)                                                    BSD General Commands Manual                                                    LOOK(1)

NAME
     look — display lines beginning with a given string

SYNOPSIS
     look [-bdf] [-t termchar] string [file ...]
...
```

**Comment by dbrgn** [comment]
I'm not sure if this is a good idea, since manpages and tldr pages have different goals (short summary vs detailed documentation). It would also be tricky to determine how the manpages should interact with the features we already have, like coloring, themes, paging, etc.

I'll keep this issue open for now. If a lot of people have interest in this, please show it with a 👍🏻 reaction on the issue or a comment.

**Comment by dflock** [comment]
Basically, this:

```bash
tldr "$@"
readonly rc="$?"
if [ "$rc" -gt 0 ]; then
  read -n 1 -p "Show manpage (y/n)?" choice
  case "$choice" in 
    y|Y ) man "$@";;
  esac
fi
```

**Comment by dbrgn** [comment]
Yeah, that manpage wrapper looks like a great idea 🙂
